lib/hocon/impl/parseable.rb

```diff
@@ -125,7 +125,7 @@ def self.force_parsed_to_object(value)
       raise Hocon::ConfigError::ConfigWrongTypeError.with_expected_actual(value.origin,
                                                          "",
                                                          "object at file root",
-                                                         value.value_type.value_type_name)
+                                                         Hocon::ConfigValueType.value_type_name(value.value_type))
```

Comment:
In this file, there is no `require` statement that ensures that `config_value_type.rb` is loaded.  It so happens that it is probably implicitly loaded by some of the other files that are `require`d by this namespace, but it's a best practice to go ahead and explicitly `require` any files that contain class/module definitions that you are relying on in your code.  So I'd go ahead and add the `require` to this file.